Products/PluggableAuthService/plugins/DelegatingMultiPlugin.py

```diff
@@ -215,7 +213,7 @@ def enumerateUsers(self, id=None, login=None, exact_match=0, sort_by=None,
 
 
 classImplements(DelegatingMultiPlugin, IDelegatingMultiPlugin,
-                IAuthenticationPlugin, IUserEnumerationPlugin, IRolesPlugin,
```

Comment:
Why was this taken out?

Reply:
The plug-in does not fulfil the whole interface. It does not have the methods `updateUser` and `updateEveryLoginName`.

Reply:
Argh. I didn't even realized those methods were part of the interface. They were put in by Maurits van Rees 6(!) years ago, but he neglected to update all built-in plugins that claim to be `IUserEnumerationPlugin`.

Maurits' merge commit is at https://github.com/zopefoundation/Products.PluggableAuthService/commit/9a26e5f53e6148011533a43948a4380770720b5b.

I'm not going to clean this up, so your stuff is fine.